docs/source/get-started/adaptive-batching.rst

```diff
@@ -94,7 +94,7 @@ Other available parameters for adaptive batching:
 
 - ``batch_dim``: The batch dimension for both input and output, which can be a tuple or a single value. See :ref:`reference/bentoml/sdk:Service api` for more information.
 - ``max_batch_size``: The upper limit for the number of requests that can be grouped into a single batch. Set this parameter based on the available resources, like memory or GPU, to avoid overloading the system.
-- ``max_latency_ms``: The maximum time in milliseconds that a batch will wait to accumulate more requests before processing.
+- ``max_latency_ms``: The total acceptable SLA (Service Level Agreement) for the end to end batch processing time. The dispatcher will respect this value by predicting the time it takes to process the batch.
```

Comment:
Why do we mention SLA here? it's a formal agreement that defines the expected level of service, right? it sounds like meeting some strict, customer-facing latency guarantee.

What about the following?

`The upper limit, in milliseconds, for the end-to-end batch processing delay. The dispatcher will respect this value by predicting the time it takes to process the batch.`

Reply:
Sure!